autopush/websocket.py

```diff
@@ -327,9 +330,27 @@ def _check_router(self, paused=False, bridge_register=None):
             self.sendMessage(json.dumps(msg).encode('utf8'), False)
             return
         # User exists?
+        # Attempt to register the user for this session
+        # First, check to see if this uaid is registered somewhere else
+        record = self.ap_settings.router.get_uaid(self.uaid)
+        if record:
+            killUrl = record.get("node_id")
+            # Only delete foreign registers. We can handle the others.
+            if killUrl != self.ap_settings.router_url:
+                log.msg("Killing duplicate uaid: %s at %s" %
+                        (self.uaid, killUrl))
+                killUrl = killUrl + "/notif/" + self.uaid
+                d = self.ap_settings.agent.request(
+                    "DELETE",
+                    killUrl.encode("utf8"),
```

Comment:
Delete call should probably include timestamp of the record as well, just in case we start executing here, and the client jumps while we're doing this and lands there. That way we only delete the client if its connect time matches the record.